src/structlog/dev.py

```diff
@@ -174,34 +188,36 @@ class ConsoleRenderer:
        `structlog.processors.format_exc_info` processor together with
        `ConsoleRenderer` anymore! It will keep working, but you can't have
        pretty exceptions and a warning will be raised if you ask for them.
+    .. versionchanged:: 21.3 The colors keyword now defaults to True on
+       non-Windows systems, and either True or False in Windows depending on
+       whether colorama is installed.
     """
 
     def __init__(
         self,
         pad_event: int = _EVENT_WIDTH,
-        colors: bool = _has_colorama,
+        colors: bool = _use_colors,
         force_colors: bool = False,
         repr_native_str: bool = False,
         level_styles: Optional[Styles] = None,
         pretty_exceptions: bool = (better_exceptions is not None),
     ):
-        self._force_colors = self._init_colorama = False
         styles: Styles
-        if colors is True:
-            if colorama is None:
-                raise SystemError(
-                    _MISSING.format(
-                        who=self.__class__.__name__ + " with `colors=True`",
-                        package="colorama",
-                    )
-                )
-
+        if colors:
             if _IS_WINDOWS:  # pragma: no cover
-                _init_colorama(self._force_colors)
-            else:
-                self._init_colorama = True
-                if force_colors:
-                    self._force_colors = True
+                # On Windows, we can't do colorful output without colorama.
+                if colorama is None:
+                    classname = self.__class__.__name__
+                    raise SystemError(
+                        _MISSING.format(
+                            who=classname + " with `colors=True`",
+                            package="colorama",
+                        )
+                    )
+                else:
+                    # Colorama must be init'd on Windows, but must NOT be
+                    # init'd on other OSes, because it can break colors.
+                    _init_colorama(force_colors)
```

Comment:
Since the `if` ends with a `raise`, drop the `else`. Then  inline `_init_colorama` since – as far as I can see – it's now only used here.. That should fix your coverage problems (the whole branch is tagged as `no cover`).